charts/cluster-scanner/values.yaml

```diff
@@ -344,3 +344,6 @@ ssl:
 
 # Option to make it possible to disable platform services
 disablePlatformScanning: false
+
+# Set Cluster Scanner pod labels
+labels: {}
```

Comment:
@maratsal thx for your contribution!

My suggestion is to use `podLabels` instead od `labels` as term, this fit with the "standard" suggested by helm on chart creations:

 *  https://github.com/helm/helm/blob/main/pkg/chartutil/create.go#L133

and used by other charts:

 * https://github.com/search?q=podLabels+language%3AYAML+path%3Avalues.yaml&type=code

Reply:
sure, will update it. used this construction just because couple of other charts in our repository is using the same pattern.